docs/brain/signals/order.md

```diff
@@ -43,6 +44,19 @@ When you will pronounce "say hello", it will trigger both synapses. To prevent t
 - **normal**: Will match if all words are present in the spoken order.
 - **strict**: All word are present. No more word must be present in the spoken order.
 - **ordered-strict**: All word are present, no more word and all word are in the same order as defined in the signal.
+- **not-contain**: The list of the given words in words-in-order will prevent that this synapse get execute if you call another synapse with a similar order which contains these words.
```

Comment:
Why `if you call another synapse with a similar order which contains these words.` ? 
Does it means that if not other synapse is matching the `not-contain` is not applied ?

Reply:
It’s a bit confusing I agree, is the following description explain it a bit better?
 
"The list of the given words in the excluded_words parameter will prevent this synapse to get execute if another synapse is called which contains the same order plus the excluded words."

Reply:
What about : 
"It will match only if all the words are present in the spoken order **WITHOUT** the `excluded_words` "

Reply:
Alright, that's sounds better, I will do the changes.